samza-azure/src/main/java/org/apache/samza/system/eventhub/EventHubConfig.java

```diff
@@ -143,6 +153,34 @@ public String getStreamEntityPath(String systemName, String streamName) {
             "EntityPath", systemName, streamName);
   }
 
+  /**
+   * Get the number of client threads, This is used to create the ThreadPool executor that is passed to the
```

Comment:
I meant that this should explain what this is used _for_ in the client. I.e., why would I want to override this?
Also, we should update the configuration reference with these new values.

Reply:
This is event hubs configuration. Update the configuration reference which points to the event hubs documentation.